server/server.go

```diff
@@ -39,7 +39,13 @@ func (cs *cribbageServer) NewRouter() http.Handler {
 	}
 
 	router.GET(`/game/:gameID`, cs.ginGetGame)
-	router.GET(`/player/:username`, cs.ginGetPlayer)
+
+	// Simple group: player
+	player := router.Group(`/player`)
+	{
+		player.GET(`/:username`, cs.ginGetPlayer)
+		player.GET(`/:username/activeGames`, cs.ginGetActiveGamesForPlayer)
```

Comment:
I'd argue that getting active games should belong to a `game` endpoint group, perhaps using a query param to indicate which player we're asking about, since games are the real resource we're trying to interact with

Reply:
I could be talked into this. :loading: